spec/factories/service_providers.rb

```diff
@@ -8,4 +8,11 @@
     return_to_sp_url '/'
     agency 'Test Agency'
   end
+
+  factory :service_provider_request do
+    uuid { SecureRandom.uuid }
+    issuer { 'test issuer name' }
+    loa { 1 }
```

Comment:
To avoid confusion and for better accuracy, can we make this a string column and maybe use a default value of `Saml::Idp::Constants::LOA1_AUTHN_CONTEXT_CLASSREF` or `http://idmanagement.gov/ns/assurance/loa/1`, which is what the column would contain.